db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20200506223904) do
+ActiveRecord::Schema.define(version: 20200512205827) do
 
   # These are extensions that must be enabled in order to support this database
```

Comment:
This was a merge conflict but wasnt sure if deleting it was the best option. Otherwise version above is from mine

Reply:
agreed, not sure. I'm afraid to touch the schema